MMCore/MMCore.h

```diff
@@ -625,6 +684,28 @@ class CMMCore
    std::vector<std::string> getLoadedPeripheralDevices(const char* hubLabel) throw (CMMError);
    ///@}
 
+   /** \name Image metadata. */
+   ///@{
+
+   /**
+    * Unclear why this is needed, deprecate?
+    */
+   bool getIncludeSystemStateCache() { 
+      return imageMDIncludeSystemStateCache_;
+   }
+
+   /**
+    * @deprecated Use setIncludeImageMetadata("SystemStateCache", state) instead
+    */
+   void setIncludeSystemStateCache(bool state) {
+      imageMDIncludeSystemStateCache_ = state;
+   }
+
+   void setIncludeImageMetadata(std::string & category, bool include) throw (CMMError);
+   ///@}
```

Comment:
I think it would be easier to use an integer bitmask rather than individual strings for this. There also needs to be a `get` method, so that an application can save and restore state -- that only works with a bitmask because the app won't know the full set of groups (because more may be added in the future).

Reply:
done